srcpkgs/libqb/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'libqb'
+pkgname=libqb
+version=1.0.3
+revision=1
+hostmakedepends="glib-devel pkg-config"
+short_desc="libqb is a library providing high performance logging, tracing, ipc, and poll"
+build_style=gnu-configure
+maintainer="Kevin Klopfenstein <kk@sudo-i.net>"
+license="LGPL-2.1"
```

Comment:
please use the SPDX identifier for the license, either -or-later or -only is missing.

Check the source code of the program for license headers.